.github/workflows/pypi-release.yml

```diff
@@ -2,157 +2,211 @@
+    needs: [sanitize-package-name, check-package-exists-pypi]
+    if: >-
+      needs.check-package-exists-pypi.outputs.exists == 'false' &&
+      inputs.test_run == 'true'
+
+    environment: PypiTest
+
+    steps:
+      - name: "Download Build Artifact - ${{ inputs.version_number }}"
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.version_number }}
+          path: .
 
+      - name: "Publish ${{ needs.sanitize-package-name.outputs.name }} v.${{ inputs.version_number }} To Test PyPI"
+        uses: pypa/gh-action-pypi-publish@v1.6.4
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+
+  prod-pypi-release:
+    runs-on: ubuntu-latest
+    needs: [sanitize-package-name, check-package-exists-pypi]
+    if: >-
+      needs.check-package-exists-pypi.outputs.exists == 'false' &&
+      inputs.test_run == false
+
+    environment: PypiProd
+
+    steps:
+      - name: "Download Build Artifact - ${{ inputs.version_number }}"
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.version_number }}
+          path: .
+
+      - name: "Publish ${{ needs.sanitize-package-name.outputs.name }} v.${{ inputs.version_number }} To PyPI"
+        uses: pypa/gh-action-pypi-publish@v1.6.4
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+  validate-package-available-pypi:
+    runs-on: ubuntu-latest
+    needs: [sanitize-package-name, test-pypi-release, prod-pypi-release]
+    # always run this step because one of the needs are always skipped.
+    if: always()
+
+    steps:
+      - name: "Audit Version And Parse Into Parts"
+        id: semver
+        uses: dbt-labs/actions/parse-semver@users/alexander-smolyakov/parse-semver-fix-warnings
+        with:
+          version: ${{ inputs.version_number }}
+
+      - name: "Fetch PyPI Info For ${{ needs.sanitize-package-name.outputs.name }} Package"
+        id: pypi_info
+        uses: dbt-labs/actions/py-package-info@users/alexander-smolyakov/py-package-info-add-ability-to-select-package-index
+        with:
+          package: ${{ needs.sanitize-package-name.outputs.name }}
+          version: ${{ steps.semver.outputs.version }}
+          check-test-index: ${{ inputs.test_run }}
+
+      - name: "Validate PyPI Info"
+        id: is-version-available
+        run: |
+          is_available=false
+          if [[ ${{ steps.pypi_info.outputs.version }} == ${{ steps.semver.outputs.version }} ]]
+          then
+            is_available=true
+          fi
+          echo "is_available=$is_available" >> $GITHUB_OUTPUT
+
+      - name: "Set Workflow Status"
+        run: |
+          title="Availability Validation"
+          if [[ ${{ steps.is-version-available.outputs.is_available }} == true ]]
+          then
+            message="The ${{ needs.sanitize-package-name.outputs.name }} v.${{ steps.semver.outputs.version }} version available in PyPI."
```

Comment:
```suggestion
            message="The ${{ needs.sanitize-package-name.outputs.name }} v${{ steps.semver.outputs.version }} version available in PyPI."
```

Reply:
Fixed in 36c4324845c575d0a2df0afb6b03840434914e61